app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,7 @@
+class TweetsController < ApplicationController
+  def profile
+    @tweets = Tweet.order(:created_at)
+
+    render("profile")
```

Comment:
`render :profile` reads better.

Reply:
depends on the rubocop config

Reply:
Actually, you can just remove this line altogether. Rails controller actions automatically call `render :name_of_action`, so you don't need to do it explicitly.

So you can just go with

```
def index
  @tweets = Tweet.latest
end
```